### ISSUE TITLE
Experimental results

### ISSUE THREAD

**Issue by shiyuan7** [Westlake-AI/A2MIM#2]
Thank you for your excellent work!

I encountered some issues while replicating your experimental results. I trained the ResNet (A3) network with random initial weights, and the top-1 accuracy is 78.196, which matches the results in Table 2. However, when I pre-trained the ResNet following the example code:

`PORT=29500 bash tools/dist_train.sh configs/openmixup/pretrain/a2mim/imagenet/r50_l3_sz224_init_8xb256_cos_ep300.py 8
`

and then trained the ResNet (A3) network with the pre-trained weights, I obtained a top-1 accuracy of 77.978, which differs from the reported accuracy of 78.9 in Table 2. I'm not sure if there are any issues with my pre-training process. Below are my pre-training log and training log with the pre-trained weights.
[pretrain.log](https://github.com/Westlake-AI/A2MIM/files/13972052/pretrain.log)
[train_withpretrain.log](https://github.com/Westlake-AI/A2MIM/files/13972053/train_withpretrain.log)

**Comment by Lupin1998** [comment]
Hi, @shiyuan7. Thanks for your question and interest in our work. Sorry for the late reply! As for your issue on the reproduction of 300-epoch pre-training with A2MIM and ResNet-50, there might be two reasons: (1) RSB A3 fine-tuning setting is unstable and might show bad results. You might try the RSB A2 setting to fine-tune the pre-trained models. We have provided relevant fine-tuned logs for your reference: [RSB A2 log](https://github.com/Westlake-AI/openmixup/releases/download/a2mim-in1k-weights/a2mim_r50_l3_sz224_init_8xb256_cos_ep300_ft_rsb_a2.log.json), [RSB A3 log](https://github.com/Westlake-AI/openmixup/releases/download/a2mim-in1k-weights/a2mim_r50_l3_sz224_init_8xb256_cos_ep300_ft_rsb_a3.log.json). (2) You may check our [pre-training log](https://github.com/Westlake-AI/A2MIM/files/14017940/a2mim_r50_l3_sz224_init_8xb256_cos_ep300.log) to verify whether there is something different. To facilitate your reproduction, we released full pre-trained models and fine-tuning results (logs/models) for A2MIM and SimMIM with ViT-S/B/L, ResNet-50/101/152/200, and ConvNeXt-S/B backbones. These files can also be downloaded from [a2mim-in1k-weights](https://github.com/Westlake-AI/A2MIM/releases/tag/a2mim-in1k-weights), [OpenMixup-a2mim-in1k-weights](https://github.com/Westlake-AI/openmixup/releases/tag/a2mim-in1k-weights) or **Baidu Cloud**: [A2MIM (3q5i)](https://pan.baidu.com/s/1aj3Lbj_wvyV_1BRzFhPcwQ?pwd=3q5i). Feel free to ask me if there are more questions. Hoping you will tackle this issue soon!

**Comment by Lupin1998** [comment]
Thanks again for your question! If there are no more problems, I will close this issue. Feel free to open a new one or contact us with new concerns.
